ratpack-codahale-metrics/src/test/groovy/ratpack/codahale/metrics/MetricsSpec.groovy

```diff
@@ -73,6 +73,7 @@ class MetricsSpec extends RatpackGroovyDslSpec {
         .jmx()
         .csv { it.reportDirectory(reportDirectory.root).reporterInterval(Duration.ofSeconds(1)) }
         .console { it.reporterInterval(Duration.ofSeconds(1)) }
+        .slf4j { it.reporterInterval(Duration.ofSeconds(1)).prefix("test.") }
```

Comment:
We need test assertions to make sure this has worked.  It should output to the `output` variable I think as Gradle configures simple logger for SLF4J which outputs to system out.  We should be able to identify this output via the prefix